contracts/precompiles/Keccak256.yul

```diff
@@ -85,7 +85,7 @@ object "Keccak256" {
                 0                                 // per precompile interpreted value (0 since circuit doesn't react on this value anyway)
             )
             // 4. Calculate number of required hash rounds per calldata
-            let numRounds := div(add(ptrLength, sub(BLOCK_SIZE(), 1)), BLOCK_SIZE())
+            let numRounds := add(div(ptrLength, BLOCK_SIZE()), 1)
```

Comment:
Note that this constant no longer represents the actual number of rounds. It is better to add a constant on why did we change it